.codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        only_pulls: true
+comment: false
```

Comment:
this is IMO useful, provides quick overview on what's changed in a comment

Reply:
I am fine with keeping them in, sure